3D Chess/Assets/Scripts/GameState.cs

```diff
@@ -95,4 +97,20 @@ private PieceEntity FindPieceEntity(Vector3 position)
                          position.x, position.y, position.z)
                  .FirstOrDefault();
     }
+
+    private async Task<Realm> CreateRealmAsync()
+    {
+        var app = App.Create("3d_chess-sjdkk");
+
+        // For this example we can just create a new random user to play the game since all users access the same game.
+        // They have to be distinct though, so anonymous credentials are not an option.
```

Comment:
Why are anonymous credentials not an option here?

Reply:
I remember reading through a Slack thread advising to not log in multiple anonymous users. Maybe that was just for a special corner case though and not a general advise.

It did - however - change it to anonymous to try it out and ended in a situation where the game crashes as soon as the second user logs in (again one of those amazing crashes, where the whole Unity editor just completely dies).

If this is supposed to work this would probably a good example and have the Zoom session we talked about so that I can learn more about how we debug those crashes. 👍

Reply:
Yeah, we should take a look at this one - it's concerning. I can also explain a little how anonymous users work.

Reply:
A conversation about anonymous users was started by me when working with a customer. I asked in slack and from the conversation basically it came out that there aren't anonymous user-**s**. There is 1 anonymous user which is always the one given back no matter how many times you log-in with anonymous credentials. I'll search for the thread and share in the dotnet group.

Reply:
I have addressed all this in b2b018b.